core/templates/base-components/base-content.component.html

```diff
@@ -103,9 +103,9 @@
   </div>
 </main>
 <style>
-  @media screen and (min-width: 600px) {
-      div.flex-display {
-    display: flex;
+      @media screen and (min-width: 600px) {
+          div.flex-display {
```

Comment:
Deindent these two lines back. This is how it should look like:
```
<style>
  @media screen and (min-width: 600px) {
    div.flex-display {
      display: flex;
    }
  }
</style>
```

Reply:
@srijanreddy98 i have done these changes can you review now ?